lasso-scm_comp/README_wa_subsets.txt

```diff
@@ -0,0 +1,68 @@
+The various *wa_subsets.ncl programs are intended to compare LASSO LES subsets
+prepared by Wayne Angevine (wayne.m.angevine@noaa.gov, CIRES at NOAA/CSL)
+to output from the 2018 CCPP SCM LASSO cases.
+
+The LES output originates from the publicly-available LASSO LES runs. Wayne
+took these data, horizontally averaged them, and reduced the number of output
+variables so that a negligible amount of disk space is required to store them.
+Wayne gave permission to share these data with other DTC staff. They are 
+available from HPSS at:
+/ESRL/BMC/ufs-phys/3year/Evan.Kalina/lasso_wa_subsets/wrfstat_extracted_s02_2018.tar 
+
+The various programs and their purposes are as follows:
+* LASSO_wrapper.ncl - Run the *wa_subsets.ncl programs for each case date
+  and output files containing means and standard deviations across all cases. 
+* LASSO_cld_wa_subsets.ncl - Analyze cloud water mixing ratio and cloud
+  fraction (time-height cross sections).
+* LASSO_pblh_wa_subsets.ncl - Analyze PBL height (time series).
+* LASSO_thtQ_prof_wa_subsets.ncl - Create time-averaged profiles of potential
+  temperature, water vapor mixing ratio, wind speed, and the physics and
+  dynamics tendencies of these variables. The averaging windows are all
+  currently two hours long, but the specific windows and their durations
+  could be easily changed (vertical profiles).
+* LASSO_thtQ_wa_subsets.ncl - Analyze potential temperature and water vapor
+  mixing ratio (time-height cross sections).
+* LASSO_wind_wa_subsets.ncl - Analyze wind speed, including at 10 m.
+  Because Wayne's LASSO subsets do not contain winds at 10 m, similarity theory
+  has been used to estimate the 10-m wind speed. The stability is assumed
+  to be neutral (time-height cross sections and time series).
+* scm_press_to_hgt.ncl - A function called by the *wa_subsets.ncl programs
+  to convert the SCM data vertical coordinate from pressure to height using the
+  geopotential equation. Needed because the LASSO LES data are only provided
+  in height coordinates. Written by Wayne Angevine and adapted to NCL by
+  Evan Kalina.
+
+* LASSO_SCM_plot_composite.ncl - Generate composites of the LASSO LES and
+  CCPP SCM results from all of the cases. Not run by LASSO_wrapper.ncl.
+* LASSO_SCM_multicase_comp.ncl - Plot the results from each individual case
+  on the same plot. The computation of the potential temperature lapse rate
```

Comment:
Still, not sure about the term "potential temperature lapse rate" here... do we really have such a thing, or it's simply the static stability criteria for dry air?